client/my-sites/plans/current-plan/index.jsx

```diff
@@ -132,22 +134,29 @@ class CurrentPlan extends Component {
 					/>
 				) }
 
-				<CurrentPlanHeader
-					isPlaceholder={ isLoading }
-					title={ title }
-					tagLine={ tagLine }
-					currentPlan={ currentPlan }
-					isExpiring={ isExpiring }
-					siteSlug={ selectedSite ? selectedSite.slug : null }
-				/>
-				{ isEnabled( 'jetpack/checklist' ) &&
-					isJetpack &&
-					! isAutomatedTransfer && (
-						<Fragment>
-							<QueryJetpackPlugins siteIds={ [ selectedSiteId ] } />
-							<JetpackChecklist />
-						</Fragment>
-					) }
+				{ this.props.showThankYou ? (
+					<CurrentPlanThankYouCard
```

Comment:
No need for the feature flag check anymore?

Reply:
I don't know why I've avoided the flag in some places. Added in 1859eeef0096da035db6b39e45030345c6625ab2